tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language-subtag-registry

```diff
@@ -44844,6 +44844,12 @@ Description: Bangla
 Added: 2005-10-16
 %%
 Type: script
+Subtag: Berf
+Description: Beria Erfe
+Added: 2025-02-06
+Comments: Unicode 17 script manually added 2025-03-26
+%%
+Type: script
```

Comment:
this file is out of date  anyway - https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry has a Mar 10 update that already has this,.

Reply:
Yes. I have done this a few times, trying to limit my changes to _updating the script metadata_, and trying not to get bogged down in unrelated changes. Which is partly defeated when the spreadsheet is out of sync with the repo, but I still don't want to tackle any and all new subtags in these updates.